transformato/mutate.py

```diff
@@ -1721,7 +1721,11 @@ def _mutate_charge(
             atom.charge = atom.initial_charge * lambda_value
             logger.debug(f"New charge: {atom.charge}")
 
-        if lambda_value != 1:
+        # check to avoid compensating charges when doing asfe
```

Comment:
This check might be not the best solution....Maybe we find a better way to ensure, that this check is not performed when doing **asfe**